cron/models.py

```diff
@@ -1,6 +1,8 @@
 from django.db import models
 import urllib.parse
 
+from django.utils import timezone
```

Comment:
Remove this import and it's a good to go

Reply:
ok

Reply:
done